tools/integrations/manage.sh

```diff
@@ -208,12 +269,6 @@ cmd_publish() {
   # time building, running tests to realize this version is a duplicate.
   _error_if_tag_exists "$versioned_image"
 
-  # building the connector
-  cmd_build "$path" "$run_tests"
-
-  # in case curing the build / tests someone this version has been published.
-  _error_if_tag_exists "$versioned_image"
-
```

Comment:
These have been pulled out into different steps cmd_build, cmd_test, both of which have separate GHA steps too. Makes it easier to see where a failure happens